pagopa/io-app
# chore(IT Wallet): [SIW-520] Remove CIE info usage screen

## Short description
This PR removes the CIE info usage screen, namely `ITW_ISSUING_CIE_INFO_USAGE_SCREEN`.

## List of changes proposed in this pull request
- Removes the screen componente;
- Removes any reference in the navigator; 
- Remove any screen related locale; 
- Updates `ItwCiePinScreen.tsx` to navigate the card reader screen;

## How to test

1. Launch the app on Android;
2. Start a PID issuing flow;
3. After confirming the CIE PIN, the app should navigate to the card reader screen rather than the previous informative screen.
